src/scss/blocks/_toggle-switch.scss

```diff
@@ -14,7 +14,8 @@ $toggle-switch-decor-ratio: 0.8;
 
 .toggle-switch__input {
   opacity: 0;
-  position: absolute;
+  // Yup, the global input styles have some pretty high specificity
+  position: absolute !important;
```

Comment:
A little bit of a CSS tip: I would add `!important` *all year* rather than trying to create a higher specificity selector. Readability FTW